modules/fishhook/1.0/scr/R/run_fishhook.R

```diff
@@ -0,0 +1,64 @@
+### Run R package dNdS to determine the significantly mutated genes ###
+
+#!/usr/bin/env Rscript
+#
+# Notes:
+#   This script is intended for use with the snakefile for fishHook in LCR-modules.
+#   It uses the snakemake objects to find maf file.
+#   Additionaly, it can support the genes of targeted panel to restrict SMG search.
+#
+
+# Load packages -----------------------------------------------------------
+message("Loading packages...")
+suppressWarnings(
+  suppressPackageStartupMessages({
+    library(fishHook)
+    library(gTrack)
+    library(rtracklayer)
+  })
+)
+#load maf and convert to GRange
+maf = dt2gr(fread(snakemake@input[[2]]))
+
+#split maf to numerous tiles
+tiles = gr.tile(seqinfo(maf), snakemake@params[[1]])
+
+events = maf %Q% (Variant_Classification != 'Silent')
```

Comment:
So the silent mutations are explicitly excluded here? Can we make it optional and have in configurable?

Reply:
include_silent_mutation is added as an option